test/lib/utils/explain-dep.js

```diff
@@ -1,8 +1,16 @@
+const { resolve } = require('path')
 const t = require('tap')
-const npm = {}
-const { explainNode, printNode } = t.mock('../../../lib/utils/explain-dep.js', {
-  '../../../lib/npm.js': npm,
-})
+const { explainNode, printNode } = require('../../../lib/utils/explain-dep.js')
+const testdir = t.testdir({})
```

Comment:
Could just use the `t.testdirName` getter here, if you're not actually using the folder.

Reply:
oh nice! didn't knew about that one 😄

Reply:
oops, is that _really_ a thing?

```
<path>/main/test/lib/utils/explain-dep.js:4
const testdir = t.testdirName()
                  ^
TypeError: t.testdirName is not a function
    at Object.<anonymous> (<path>/main/test/lib/utils/explain-dep.js:4:19)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47
```

Reply:
It's a string, not a function.

Reply:
aaaah, that makes sense then!